kw/platform/monkey.py

```diff
@@ -0,0 +1,26 @@
+import importlib
+import threading
+
+
+# Define modules and which of them should be patched automatically.
+_PATCH_MODULES = {"requests": False}
+_PATCHED_MODULES = set()
+
+
+def _patch_module(module):
+    with threading.Lock():
+        if module in _PATCHED_MODULES:
+            return False
+        path = f"kw.platform.{module}.patch"
+        imported_module = importlib.import_module(path)
+        imported_module.patch()
+
+        _PATCHED_MODULES.add(module)
+        return True
+
+
+def patch(**patch_modules):
+    modules = [m for m, should_patch in patch_modules.items() if should_patch]
+    for module in modules:
+        if module in _PATCH_MODULES.keys():
+            _patch_module(module)
```

Comment:
We might need to add a copyright notice since this seems to be almost the same as the code in ddtrace.

I asked about that here https://github.com/DataDog/dd-trace-py/pull/977#issuecomment-505832227

Reply:
Are we going to wait for their answer before merging this?

Reply:
No, we don't need to.